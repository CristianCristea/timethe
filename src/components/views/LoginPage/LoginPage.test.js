/* global jest, it, expect */
import React from 'react';
import { shallow } from 'enzyme';
import { LoginPage } from './index';

it('should correctly display the login page', () => {
  const wrapper = shallow(<LoginPage startLogin={() => { }} />);
  expect(wrapper).toMatchSnapshot();
});

it('should call startLogout on buton click', () => {
  const startLogin = jest.fn();
  const wrapper = shallow(<LoginPage startLogin={startLogin} />);
  wrapper.find('button').simulate('click');
  expect(startLogin).toHaveBeenCalled();
});