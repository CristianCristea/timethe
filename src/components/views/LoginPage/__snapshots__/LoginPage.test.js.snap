// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly display the login page 1`] = `
<Container
  className="login-page"
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Jumbotron
        className="login-page__jumbotron"
        tag="div"
      >
        <h4
          className="mb-5"
        >
          Project time tracker
        </h4>
        <GoogleButton
          className="login-page__google-signin-btn"
          disabled={false}
          id="login-page__google-signin-btn"
          label="Sign in with Google"
          onClick={[Function]}
          type="dark"
        />
      </Jumbotron>
    </Col>
  </Row>
</Container>
`;
